etl/steps/data/garden/demography/2023-04-07/population_education_wittgenstein.meta.yml

```diff
@@ -0,0 +1,37 @@
+# (Inherited from meadow, remove if not different.)
+all_sources:
+  - sources: &sources
+      name: Wittgenstein Centre (2018)
+      published_by: Wittgenstein Centre for Demography and Global Human Capital (WIC) Wittgenstein Centre Data Explorer. Version 2.0 2018
+      url: http://dataexplorer.wittgensteincentre.org/wcde-v2/
+      date_accessed: 2023-04-07
+      publication_year: 2018
+      # description: Source description.
```

Comment:
If you haven't changed anything, you could simply remove this and import the source directly from snapshot and meadow.